src/main/java/de/retest/ui/descriptors/SuffixAttribute.java

```diff
@@ -23,11 +23,17 @@ public SuffixAttribute( final int value, final String variableName ) {
 		super( SUFFIX_KEY, Integer.toString( value ), variableName );
 	}
 
+	/**
+	 * @deprecated
```

Comment:
I think this leads to Javadoc warnings because it now has Javadoc, but incomplete (e.g. no `@param`).